[debug] [2019-06-29T20:00:04.182Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:00:04.186Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2019-06-29T20:00:04.187Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:00:04.188Z] Platform:      win32
[debug] [2019-06-29T20:00:04.188Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:00:04.190Z] Time:          Sat Jun 29 2019 23:00:04 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:00:04.191Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=404919334&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-06-29T20:00:09.088Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-06-29T20:04:30.413Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:04:30.415Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-06-29T20:04:30.415Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:04:30.415Z] Platform:      win32
[debug] [2019-06-29T20:04:30.416Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:04:30.417Z] Time:          Sat Jun 29 2019 23:04:30 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:04:30.417Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=722104138&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-06-29T20:04:32.688Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-06-29T20:06:04.941Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:06:04.944Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --no-localhost
[debug] [2019-06-29T20:06:04.944Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:06:04.944Z] Platform:      win32
[debug] [2019-06-29T20:06:04.944Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:06:04.945Z] Time:          Sat Jun 29 2019 23:06:04 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:06:04.945Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=188184740&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob
[info] 
[debug] [2019-06-29T20:06:07.397Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-06-29T20:08:00.656Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:08:00.658Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --interactive
[debug] [2019-06-29T20:08:00.659Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:08:00.659Z] Platform:      win32
[debug] [2019-06-29T20:08:00.659Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:08:00.660Z] Time:          Sat Jun 29 2019 23:08:00 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:08:00.660Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=159777307&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-06-29T20:08:02.835Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-06-29T20:10:42.728Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:10:42.730Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2019-06-29T20:10:42.730Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:10:42.730Z] Platform:      win32
[debug] [2019-06-29T20:10:42.730Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:10:42.731Z] Time:          Sat Jun 29 2019 23:10:42 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:10:42.731Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=316122137&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-06-29T20:10:44.901Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2019-06-29T20:12:33.659Z] ----------------------------------------------------------------------
[debug] [2019-06-29T20:12:33.663Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Desali Gdalo\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2019-06-29T20:12:33.664Z] CLI Version:   7.0.2
[debug] [2019-06-29T20:12:33.664Z] Platform:      win32
[debug] [2019-06-29T20:12:33.665Z] Node Version:  v10.16.0
[debug] [2019-06-29T20:12:33.666Z] Time:          Sat Jun 29 2019 23:12:33 GMT+0300 (GMT+03:00)
[debug] [2019-06-29T20:12:33.666Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=115912605&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2019-06-29T20:12:48.310Z] Error: spawn cmd ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[error] 
[error] Error: An unexpected error has occurred.
